DoSomethingArchive/legacy-website
# Fixes #5162: Copy obeys user's lang over URL

#### What's this PR do?

Addresses inconsistent behavior where a logged-in user's profile language could override the language indicated by the URL prefix.
#### Where should the reviewer start?

This is just a config update to the `dosomething_global` feature, specifically for the language detection settings (`admin/config/regional/language/configure`).
#### How should this be manually tested?
1. Log in.
2. Load a campaign in multiple languages, each in its own tab.
3. You should see the appropriate language for all copy (given that the translation exists).
4. Change your user's preferred language.
5. Reload the various campaign tabs.
6. You should still see the appropriate language in each tab, based on the language indicated by the URL.
#### Any background context you want to provide?

N/A
#### What are the relevant tickets?
#5162
#### Screenshots (if appropriate)

N/A
#### Questions:
- Does the knowledge base need an update? _No_
